tst/ProducerFunctionalityTest.cpp

```diff
@@ -1029,7 +1034,7 @@ TEST_F(ProducerFunctionalityTest, basic_reset_connection)
     EXPECT_LT(0, mStreamClosedFnCount);
     EXPECT_LT(0, mFragmentAckReceivedFnCount);
     EXPECT_LT(totalFragments, mPersistedFragmentCount);
-    EXPECT_LT(4, mPutStreamFnCount);
+    EXPECT_EQ(3, mPutStreamFnCount);
```

Comment:
Looks like this ends up being 4 in travis CI for MacOS.  Perhaps we should change this to:

`EXPECT_LE(4, mPutStreamFnCount)`

Reply:
It’s a timing issue again. Let me see if I can be more precise and if not I will do LE